assets/clickhouse_native_json/main.py

```
@@ -0,0 +1,174 @@
+
+        if not manual_column_names:
+            self.queries = self.config["queries_automatic_json"]
+
+    @property
+    def mount_points(self):
+        return {
+            f"{self.script_dir}/include/config.xml": "/etc/clickhouse-server/config.d/benchconfig.xml",
+            f"{self.script_dir}/include/users.xml": "/etc/clickhouse-server/users.d/benchconfig.xml",
+        }
+
+    def clickhouse_execute(self, query):
```

Ruff (0.12.2)

Missing return type annotation for public function `clickhouse_execute`

(ANN201)